tests/btc/transactions.test.ts

```diff
@@ -724,7 +706,8 @@ describe('bitcoin transactions', () => {
 
     const filteredUnspentOutputs = utxos.filter((unspentOutput) => {
```

Comment:
Should we reuse the `filterUtxos` function from `transactions/btc.ts` file here?